Sources/Web3Core/Transaction/CodableTransaction.swift

```diff
@@ -271,7 +271,7 @@ extension CodableTransaction: CustomStringConvertible {
         var toReturn = ""
         toReturn += "Transaction" + "\n"
         toReturn += String(describing: self.envelope)
-        toReturn += "from: " + String(describing: self.sender?.address)  + "\n"
+        toReturn += "from: " + String(describing: self.sender)  + "\n"
```

Comment:
Would this also catch if `from` is set instead of `sender`?

Reply:
I didn't get it, could you explain in more detail?

Reply:
@albertopeam what I meant in #766 that tx.description is not returning anything for `from` if it's set.

So I just wanted to make sure that sender may return this.

I guess an easy test would be create a tx object, set an address to `from`. Check if tx.description returns that address.

Reply:
Added the test scenario, not possible to mutate from field for a transaction. Take a look please, if you can specify a better way I will take a look. Thanks :)